format/index.md

```diff
@@ -593,10 +594,9 @@ Data, including resources and relationships, can be fetched by sending a
 `GET` request to an endpoint.
 
 Clients **MUST** indicate that they can accept the JSON API media type, per
-the semantics of the HTTP `Accept` header. If present, this header value **MUST**
-also include any media type extensions relevant to the request. Servers **MUST**
-return a `406 Not Acceptable` status code if this header specifies an
-unsupported media type.
+the semantics of the HTTP `Accept` header. Servers **MUST** return a
+`406 Not Acceptable` status code if this header specifies only unsupported
+media types.
```

Comment:
**Edit**: I've now moved this into the top section, as you suggested. About the other logic, seem my line notes on the new diff.